client/src/App.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
+import Signup from './pages/signUp';
+import './App.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
 
 function App() {
   return (
-    <div className="App">
-      <h1>Taskillo</h1>
+    <div className="container">
```

Comment:
Please, don't delete the general code.
```suggestion
    <div className="App container">
```